README.md

```diff
@@ -1,2 +1,38 @@
-# Bright Futures Academy
-http://brightfuturesatl.org/the-program/academy/
+# Bright Futures Academy Parent Portal
+This is a web app built to manage payroll at [Bright Futures Academy](
+http://brightfuturesatl.org/the-program/academy/) in Atlanta.
+It provides information for prospective students, and enrolled families
+can sign in to manage their students' tuition and other payments. 
+
+## Getting Started
+### Prerequisites
+You'll need [npm](https://nodejs.org/en/) to run this project.
+
+### Installing
+First, install the packages for both the client and server.
+```
+npm run postinstall
+```
+
+## Running the app
+You can start a development server like this:
+```
+npm run server
+```
+Or you can run the full app like this:
+```
+npm run dev
+```
+
```

Comment:
If you want to just run the front end then `cd client` and `npm start`.

Reply:
Oh also don't forget to mention running `npm install`

Reply:
`npm run postinstall` covers this